src/lib/Libutil/misc_utils.c

```diff
@@ -2163,12 +2163,12 @@ crc_file(char *filepath)
 
 /**
  * @brief
- * 	Return the maximum number of servers possible in the cluster
+ * 	Return multiserver mode
  *
  * @return int
  */
 int 
-get_max_servers(void)
+get_msvr_mode(void)
 {
-	return 1;
+	return 0;
```

Comment:
We need it to test - this was Nithin's earlier code that disabled sending IS_CLUSTER_ADDRS in case of multiserver configuration. This is used in node_manager.c